news/2022-03-15-news-announcing-three-new-board-members.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: news
+title: Announcing three new Rust Foundation Board Members
+description: New appointments further strengthen Foundation's expert roster
+tags:
+  - foundation
+  - press
+  - membership
+---
+
+The [Rust Foundation](https://foundation.rust-lang.org) is delighted to announce that three new individuals have joined the Board of Directors.
+
+## Ryan Levick (Project Director - Core Team)
```

Comment:
@abibroom Just want to make sure that it was intentional that we did not put the photos of each new board member in the blog post?

Reply:
I guess we could include the photos - I assume that's easy enough for you to add in?